job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -44,4 +44,5 @@ class Constants {
     static final String CEKIT_CACHE_LOCAL = ""
     static final String VERBOSE	= "false"
     static final String LOCAL_MVN_REP = "/home/jenkins/.m2/repository"
+    static final String KOGITO_PREFIX = "0.7.0"
```

Comment:
why we hardcoding the version of kogito here?

Reply:
this is only a prefix - this will change each time we do an upgrade.

Reply:
exactly my concern is to avoid manual update,

Reply:
@mareknovotny  the Constants are upgraded since ever - every time a version changes the version has to change here also - all scripts we have take constants - they only have to be modified here. I don't expect these changes can be done automatically. Somewhere this has to be changed.